pages/Elevator-Plate.md

```diff
@@ -0,0 +1,7 @@
+## Obtaining
+The Elevator Plate can be crafted in an [Enchanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+Elevator Plates are automatically connected when sharing the X and Z coordinates.
```

Comment:
I think this sentence is a little confusing. Maybe write that they are considered "connected" as long as they are placed along the same Y axis, meaning they need to be above/beneach one another.